### PR TITLE
New package: BastianPedersen.lazydocker version 0.23.1

### PR DESCRIPTION
**Pull request by bassepeder**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/134111)

### CODE REVIEW

**Comment by bastianpedersen** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
image

**Comment by bastianpedersen** [comment]
> Manual Validation ended with:
> 
> > 2024/01/15 10:47:21 > could not determine host context "default" does not exist 2024/01/15 10:47:21 unable to parse docker host ``
> 
> (Automated response - build 566.)

Hi. This is a known issue and will be fixed in the [root repository](https://github.com/jesseduffield/lazydocker). Once the fix is deployed I will re-upload with the new binaries.

**Comment by bastianpedersen** [comment]
> image

This has nothing to do with lazydocker. It is something wrong with your Docker configuration.

**Comment by stephengillie** [comment]
Hi @bastianpedersen,

I have some items out of order here. Let me explain:

1. This package installs and launches normally in a Windows 10 VM.
2. This package gives the Error from [my 2nd message](https://github.com/microsoft/winGet-pkgs/pull/134111#issuecomment-1892641731). Does this mean Docker is a dependency? 
3. When trying with `Docker.DockerDesktop` as a dependency, I got the error from [my first message](https://github.com/microsoft/winGet-pkgs/pull/134111#issuecomment-1892641061).

**Comment by bastianpedersen** [comment]
> Hi @bastianpedersen,
> 
> I have some items out of order here. Let me explain:
> 
> 1. This package installs and launches normally in a Windows 10 VM.
> 2. This package gives the Error from [my 2nd message](https://github.com/microsoft/winGet-pkgs/pull/134111#issuecomment-1892641731). Does this mean Docker is a dependency?
> 3. When trying with `Docker.DockerDesktop` as a dependency, I got the error from [my first message](https://github.com/microsoft/winGet-pkgs/pull/134111#issuecomment-1892641061).

Yes, Docker is a core dependency for this application. Hence the name `lazydocker` 😅 . Please make sure you have clean install of Docker Desktop on your system.

File: manifests/b/BastianPedersen/lazydocker/0.23.1/BastianPedersen.lazydocker.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BastianPedersen.lazydocker
+PackageVersion: 0.23.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: lazydocker.exe
+  PortableCommandAlias: lazydocker
```
Comment (stephengillie) [inline manifests/b/BastianPedersen/lazydocker/0.23.1/BastianPedersen.lazydocker.installer.yaml:10]:
```suggestion
  PortableCommandAlias: lazydocker
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Docker.DockerDesktop
```

Comment (bastianpedersen) [reply manifests/b/BastianPedersen/lazydocker/0.23.1/BastianPedersen.lazydocker.installer.yaml:10]: Thank you very much! I was unaware that this option existed.

**Comment by bastianpedersen** [comment]
Can I please get some help here? The application installs just fine on my Windows 11 build.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334972

(Automated response - build 599.)

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Thanks for your patience.

**Comment by darkvertex** [comment]
This PR adds noise with a new package ID for a package that already existed in the system as `JesseDuffield.Lazydocker`... Now the same product exists twice with two names (`BastianPedersen.lazydocker` + `JesseDuffield.Lazydocker`) and for the same version, referencing the same installer.😞

PR #135365 is the correct version of what this should have been, adding a new version to the existing package ID instead of registering a new package.

@stephengillie Does the winget repo ever do retractions for cases of duplicates like these?

**Comment by stephengillie** [comment]
Thank you for pointing this out. 
- https://github.com/microsoft/winget-pkgs/pull/155072

**Pull request merged**